### DISCUSSION TITLE
Next steps for lab manuals

### DISCUSSION THREAD

**Discussion by Kourtney-Burger** [nmfs-ost/PAM-OpenScience#27] [General]
Explain how you and your team will continue to work on your center's PAM lab manual.  

_Example: I will set aside 1.5 hours/week to continue adding info to the Lab Manual, and I will ask others on our team to do the same. We will give an update at each of our lab meetings._

**Comment by rebecca-vanhoeck** [comment]
Bella and I will continue meeting weekly to tackle the NEFSC lab manual. We will slowly work to migrate the lab manual to github up and bring the PAB into the open science community

**Reply by Isabellagarf** [reply]
Adding to this, wrapping in techniques learned in opens capes during this process and teaching lab members through our show and tell meetings (or future meetings)

**Comment by cberchok** [comment]
I will take the weekly MML Openscapes 2 hour period to actually start working on my lab manual.  It is a standing meeting that's been running for the past year.

**Comment by chomweaverNOAA** [comment]
I need to touch base with Shannon regarding the hardware page on our lab manual. Also, I need to start a repo for CalCurCEAS (not lab manual related but it will allow me to practice Openscape concepts)

**Comment by lynne-hodge** [comment]
I will continue to add information to pages 1-2 hrs/wk. I will present what I have accomplished at our lab monthly meetings and also seek out individuals for suggestions on format and content.
